metadata-ingestion/tests/integration/tableau/test_tableau_ingest.py

```diff
@@ -545,7 +545,34 @@ def test_value_error_projects_and_project_pattern(
             pipeline_config=new_config,
         )
     except Exception as e:
-        assert "projects is deprecated. Please use project_pattern only" in str(e)
+        assert "projects is deprecated. Please use project_path_pattern only" in str(e)
+
+
+def test_project_pattern_deprecation(pytestconfig, tmp_path, mock_datahub_graph):
```

Comment:
Could you please add a test-case which utilizes `project_path_pattern` for ingestion